Upstream: notified
URL: https://github.com/lupoDharkael/flameshot/pull/757
Reason: Since QT 5.15, QPainterPath needs to be explicitly included.
diff --git a/src/third-party/Qt-Color-Widgets/src/color_wheel.cpp b/src/third-party/Qt-Color-Widgets/src/color_wheel.cpp
index 335b29c..6f4d9fa 100644
--- a/src/third-party/Qt-Color-Widgets/src/color_wheel.cpp
+++ b/src/third-party/Qt-Color-Widgets/src/color_wheel.cpp
@@ -27,6 +27,7 @@
 #include <QLineF>
 #include <QDragEnterEvent>
 #include <QMimeData>
+#include <QPainterPath>
 #include "color_utils.hpp"
 
 namespace color_widgets {
